specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```diff
@@ -0,0 +1,10773 @@
+          "x-nullable": false
+        }
+      },
+      "required": [
+        "name",
+        "kind"
+      ],
+      "description": "Contains configuration options specific to the algorithm used during indexing and/or querying."
+    },
+    "HnswVectorSearchAlgorithmConfiguration": {
+      "type": "object",
+      "x-ms-discriminator-value": "hnsw",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VectorSearchAlgorithmConfiguration"
+        }
+      ],
+      "properties": {
+        "hnswParameters": {
+          "x-ms-client-name": "Parameters",
+          "$ref": "#/definitions/HnswParameters",
+          "description": "Contains the parameters specific to hnsw algorithm."
+        }
+      },
+      "description": "Contains configuration options specific to the hnsw approximate nearest neighbors algorithm used during indexing and querying. The hnsw algorithm offers a tunable trade-off between search speed and accuracy."
+    },
+    "HnswParameters": {
+      "type": "object",
+      "properties": {
+        "m": {
+          "type": "integer",
+          "format": "int32",
+          "x-nullable": true,
+          "minimum": 4,
+          "maximum": 10,
+          "default": 4,
+          "description": "The number of bi-directional links created for every new element during construction. Increasing this parameter value may improve recall and reduce retrieval times for datasets with high intrinsic dimensionality at the expense of increased memory consumption and longer indexing time."
+        },
+        "efConstruction": {
+          "type": "integer",
+          "format": "int32",
+          "x-nullable": true,
+          "minimum": 100,
+          "maximum": 1000,
+          "default": 400,
+          "description": "The size of the dynamic list containing the nearest neighbors, which is used during index time. Increasing this parameter may improve index quality, at the expense of increased indexing time. At a certain point, increasing this parameter leads to diminishing returns."
+        },
+        "efSearch": {
+          "type": "integer",
+          "format": "int32",
+          "x-nullable": true,
+          "minimum": 100,
+          "maximum": 1000,
+          "default": 500,
+          "description": "The size of the dynamic list containing the nearest neighbors, which is used during search time. Increasing this parameter may improve search results, at the expense of slower search. At a certain point, increasing this parameter leads to diminishing returns."
+        },
+        "metric": {
+          "$ref": "#/definitions/VectorSearchAlgorithmMetric",
+          "x-nullable": true,
+          "description": "The similarity metric to use for vector comparisons."
+        }
+      },
+      "description": "Contains the parameters specific to hnsw algorithm."
+    },
+    "ExhaustiveKnnVectorSearchAlgorithmConfiguration": {
```

Comment:
Is this part of the GA? I thought this was being added in the `2023-10-01-Preview`

Reply:
Right now the plan is to GA this feature (exhaustive knn) as well